### PR TITLE
[Fix]: Adds check and tests for Image url

### PR DESCRIPTION
**Pull request by shraddha-kesari**
This Adds a check for the Image url depending on the `PageType`.
If the `pageType` is `story-page-amp` then the Image url params change to be `w=1200&h=750&auto=format%2Ccompress&fit=max` orelse it will be `w=480&h=270&auto=format%2Ccompress&fit=max`

### CODE REVIEW

File: test/structured_data_tags_test.js [hunk 1]
```diff
@@ -164,9 +164,8 @@ describe('StructuredDataTags', function() {
...
...pe","url":"http://www.quintype.com/","logo":"https://quintype.com/logo.png","sameAs":["https://www.facebook.com/quintype","https://twitter.com/quintype_inc","https://plus.google.com/+quintype","https://www.youtube.com/user/Quintype"]},"author":[{"@type":"Person","givenName":"Greeshma","name":"Greeshma"}],"keywords":"","thumbnailUrl":"https://images.assettype.com/quintype-demo/2018-03/a27aafbf-8a27-4f42-b78f-769eb04655d6/efa66751-e534-4a18-8ebe-e02189c356d9.jpg?w=480&h=270&auto=format%2Ccompress&fit=max","articleBody":"","dateCreated":"2018-02-28T11:11:04Z","dateModified":"2018-04-20T06:03:25Z","name":"Personalise or perish - Why publishers need to use personalised content","isPartOf":{"@type":"WebPage","url":"https://madrid.quintype.io/politics/2018/02/28/personalise-or-perish---why-publishers-need-to-use-personalised-content","primaryImageOfPage":{"@type":"ImageObject","url":"https://images.assettype.com/quintype-demo/2018-03/a27aafbf-8a27-4f42-b78f-769eb04655d6/efa66751-e534-4a18-8ebe-e02189c356d9.jpg?w=480&h=270&auto=format%2Ccompress&fit=max","width":"480","height":"270"}},"articleSection":"Section Name","@type":"Article","@context":"http://schema.org"}</script>', storyString);
```
Comment (himani39) [inline test/structured_data_tags_test.js:167]: Hey, unable to understand how the expected string has this extra `BreadcrumbList`. It was not there before

Comment (shraddha-kesari) [reply test/structured_data_tags_test.js:167]: It was there for few other tests and now it got added to all of them.

File: src/structured-data/structured-data-tags.js [hunk 2]
```diff
@@ -33,18 +33,18 @@ function ldJson(type, fields) {
   };
 }
 
-function imageUrl(publisherConfig, s3Key) {
+function imageUrl(publisherConfig, s3Key, pageType) {
   const imageSrc = /^https?.*/.test(publisherConfig['cdn-image']) ? publisherConfig['cdn-image'] : `https://${publisherConfig['cdn-image']}`;
-  return `${imageSrc}/${s3Key}?w=480&h=270&auto=format%2Ccompress&fit=max`;
+  return pageType === 'story-page-amp' ? `${imageSrc}/${s3Key}?w=1200&h=750&auto=format%2Ccompress&fit=max`: `${imageSrc}/${s3Key}?w=480&h=270&auto=format%2Ccompress&fit=max`;
```
Comment (sharangj) [inline src/structured-data/structured-data-tags.js:38]: pageType is something this function should not worry about. Can you accept width and height as optional params instead?

File: src/structured-data/structured-data-tags.js [hunk 3]
```diff
@@ -33,18 +33,18 @@ function ldJson(type, fields) {
...
-function generateCommonData(structuredData = {}, story = {}, publisherConfig = {}) {
+
+function generateCommonData(structuredData = {}, story = {}, publisherConfig = {}, pageType = "") {
   const storyUrl = story.url || `${publisherConfig['sketches-host']}/${story.slug}`;
   const mainEntityUrl = (Object.keys(story).length > 0 && structuredData.storyUrlAsMainEntityUrl) ? storyUrl : get(structuredData, ['organization', 'url'], '');
-
   return Object.assign({},
     {'headline' : story.headline,
-    "image": [imageUrl(publisherConfig, story['hero-image-s3-key'])],
+    "image": [imageUrl(publisherConfig, story['hero-image-s3-key'], pageType)],
```
Comment (sharangj) [inline src/structured-data/structured-data-tags.js:47]: Pass the width and height based on the pageType here.

**Pull request merged**